cmake/Modules/opm-upscaling-prereqs.cmake

```diff
@@ -27,5 +27,6 @@ set (opm-upscaling_DEPS
 	dune-grid REQUIRED;
 	opm-common REQUIRED;
 	opm-core REQUIRED;
+        opm-output REQUIRED;
 	dune-cornerpoint REQUIRED"
```

Comment:
You should **probably** switch the order of -output and -cornerpoint.  The latter does not require the former.